package.json

```diff
@@ -80,9 +80,11 @@
   ],
   "dependencies": {
     "@types/hast": "^2.0.0",
+    "@types/prop-types": "^15.0.0",
     "@types/unist": "^2.0.0",
     "comma-separated-tokens": "^2.0.0",
     "hast-util-whitespace": "^2.0.0",
+    "mdast-util-to-hast": "^12.0.0",
```

Comment:
Nice catch!
Could you solve the `RemarkRehypeOptions` by instead importing it from `remark-rehype`? That’s already a dependency

Reply:
The source is correctly importing it from `remark-rehype`. I created https://github.com/microsoft/TypeScript/issues/48242 since it seems like a TypeScript bug.

Reply:
I've narrowed this PR to just add `@types/prop-types` for now. Hopefully the dependency on `mdast-util-to-hast` upstream.

Reply:
> can be fixed in upstream.

This sounds like a pnpm problem.